README.md

```diff
@@ -849,26 +849,7 @@ When a rule is triggered, the script is provided the event instance that trigger
 The specific data depends on the event type.
 The `event` object provides several information about that trigger.
 
-This tables gives an overview over the `event` object:
-| Property Name     | Trigger Types                                       | Description                                                                         | Rules DSL Equivalent   |
-|-------------------|-----------------------------------------------------|-------------------------------------------------------------------------------------|------------------------|
-| `oldState`        | `ItemStateChangeTrigger`, `GroupStateChangeTrigger` | Previous state of Item or Group that triggered event                                | `previousState`        |
-| `newState`        | `ItemStateChangeTrigger`, `GroupStateChangeTrigger` | New state of Item or Group that triggered event                                     | N/A                    |
-| `state`           | `ItemStateUpdateTrigger`, `GroupStateUpdateTrigger` | State of Item that triggered event                                                  | `triggeringItem.state` |
-| `receivedCommand` | `ItemCommandTrigger`, `GroupCommandTrigger`         | Command that triggered event                                                        | `receivedCommand`      |
-| `receivedState`   | `ItemStateUpdateTrigger`, `GroupStateUpdateTrigger` | State that triggered event                                                          | N/A                    |
-| `receivedTrigger` | `ChannelEventTrigger`                               | Trigger that triggered event                                                        | N/A                    |
-| `itemName`        | all                                                 | Name of Item that triggered event                                                   | `triggeringItem.name`  |
-| `eventType`       | all except `ThingStatus****Trigger`s                | Type of event that triggered event (change, command, time, triggered, update)       | N/A                    |
-| `triggerType`     | all except `ThingStatus****Trigger`s                | Type of trigger that triggered event (for `TimeOfDayTrigger`: `GenericCronTrigger`) | N/A                    |
-
-All properties are type of String.
-
-**NOTE:**
-`ThingStatusUpdateTrigger`, `ThingStatusChangeTrigger` use *Thing* and `ChannelEventTrigger` uses the the trigger channel name as value for `itemName`.
-`Group****Trigger`s use the equivalent `Item****Trigger` as trigger for each member.
-
-You may use [utils.dumpObject(event)](https://openhab.github.io/openhab-js/utils.html#.dumpObject) to get all properties of an `event` object.
+See [openhab-js - EventObject](https://openhab.github.io/openhab-js/rules.html#.EventObject) for documentation.

```

Comment:
So we have the table describing the event object when using UI based rules earlier in this file, and then this one which describes the event object when using files.  Is there a reason to remove this one, but not the other?  I think its nice to have here as knowing what's in the event object is pretty important and personally i ended up doing a console.dir(event) for a while in my rules to remember what was being passed in.

Reply:
I have removed the file-based event object table from the README and instead linked the new file-based event object doc in JSDoc.

Reply:
So the docs are not gone but moved.

Reply:
Right , I see that, but now we list the event object only for UI rules but not files? It seems like we should have both or none in the README.   My other point is that i see this as information anyone who is using this library will need, so maybe its best to have it in the readme vs having to read the API docs which is i think is for more advanced users?

Reply:
I see your point, I will update the file based one in the README instead of removing it.
I will keep the link to JSDoc dir more complete docs, but add back the basics to the README.

Reply:
👍